docs/io/zeta.ipynb

```diff
@@ -17,16 +17,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Recombination coefficient might be a little bit misleading.</p><p>Maybe "ground state recombination fractions" or simply "zeta data".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/carsus/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>